.internal-ci/helm/fog-test-client/values.yaml

```diff
@@ -72,4 +72,4 @@ serviceMonitor:
   enabled: true
 
 jaegerTracing:
-  collector: 'dns:///jaeger-collector:14250'
+  enabled: false
```

Comment:
this appears to be how it shows everywhere else

Reply:
ISugarswitch